cmd/maya-apiserver/spc-watcher/lease.go

```diff
@@ -49,35 +99,76 @@ type spcLease struct {
 	leaseKey string
 	// oecs is the openebs clientset
 	oecs clientset.Interface
+	// kubeclientset is a standard kubernetes clientset
```

Comment:
Please check if spcLease behaviours can be moved into earlier defined `lease` struct